core/Pos/Core/Arbitrary.hs

```diff
@@ -119,6 +121,12 @@ instance Arbitrary Types.EpochOrSlot where
         ]
     shrink = genericShrink
 
+-- | TODO: Replace with code from Test.Pos.CborSpec:arbitraryUnparsedFields
```

Comment:
Good point -- I think we should just take the plunge and remove this `TODO`, will take care of it!